SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -466,131 +510,86 @@ void USpatialActorChannel::SetChannelActor(AActor* InActor)
 	}
 
 	// Set up the shadow data for the handover properties. This is used later to compare the properties and send only changed ones.
-	USpatialInterop* Interop = SpatialNetDriver->GetSpatialInterop();
-	check(Interop);
-	USpatialTypeBinding* Binding = Interop->GetTypeBindingByClass(InActor->GetClass());
-	if (Binding)
-	{
-		const FHandoverHandlePropertyMap& HandoverProperties = Binding->GetHandoverHandlePropertyMap();
-		uint32 Size = 0;
-		for (auto& Property : HandoverProperties)
-		{
-			// Make sure we conform to Unreal's alignment requirements; this is matched below and in ReplicateActor()
-			Size = Align(Size, Property.Value.Property->GetMinAlignment());
-			Size += Property.Value.Property->GetSize();
-		}
-		HandoverPropertyShadowData.Empty();
-		HandoverPropertyShadowData.AddZeroed(Size);
-		uint32 Offset = 0;
-		for (auto& Property : HandoverProperties)
-		{
-			Offset = Align(Offset, Property.Value.Property->GetMinAlignment());
-			Property.Value.Property->InitializeValue(HandoverPropertyShadowData.GetData() + Offset);
-			Offset += Property.Value.Property->GetSize();
-		}
+	check(!HandoverShadowDataMap.Contains(InActor));
```

Comment:
Shouldn't we just check that HandoverShadowDataMap is empty here? Is there a valid use case where this is partially setup already?